Charts: update Spline Area demo description (#1980)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/SplineArea/description.md

> Spline area series unlike area series display smooth lines that connect data points. This demo shows different spline area [series types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). Use the drop-down editor below the Chart to select the type.

Reviewer: ```suggestion
Spline area series display smooth lines that connect data points. This demo shows different spline area [series types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). Use the drop-down editor below the Chart to select the type.
```

- *splinearea*    

> The spline area series draws the line between neighboring data points and fills the area under that line. If the Chart contains multiple area series, they overlap each other.

Reviewer: ```suggestion
The spline area series draws a line between neighboring data points and fills the area under the line. If the Chart contains multiple area series, they overlap.
```

- *stackedsplinearea*    

> The Chart displays areas stacked on top of each other without overlapping. This type of Chart is useful if you need to compare contributions of each series to the overall value.

Reviewer: ```suggestion
The Chart displays areas stacked on top of each other without overlapping. This type of Chart is useful if you need to compare the contribution of each series to the overall value.
```

- *fullstackedsplinearea*    

> The Chart stacks the areas. For each argument, it displays values as percentages of the total, as opposed to absolute values. The topmost series points are always plotted at 100%, and the graph fully covers the Chart's pane. 

Reviewer: ```suggestion
The Chart stacks the areas. Values are displayed as percentages of the total for each argument, as opposed to absolute values. The topmost series points are always plotted at 100%, and the graph completely fills the Chart's pane. 
```
